### PR TITLE
Signup/Login Validation 

### PR DESCRIPTION
**Pull request by beckystar4**
This is a sign up page and login page with validation for name, email, and password. 

## Trello Card

[Replace this text with a link to your trello card(s).](https://trello.com/c/okpO9WON/10-create-signup-page)
https://trello.com/c/RD17jOtX/28-password-validation

## Description

Added a signup page and the necessary validation for first_name, last_name, email, and password. Then developed tests in order to check that it was working.

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+      <%= f.label :last_name %>
+      <%= f.text_field :last_name, class: 'form-control' %>
+
+      <%= f.label :email %>
+      <%= f.email_field :email, class: 'form-control' %>
+
+    <%# turn into a drop down if possible %>
+      <%= f.label :country %> 
+      <%= f.text_field :country, class: 'form-control' %>
```
Comment (JMGbaum) [inline app/views/users/new.html.erb]: We may want to make this a dropdown list instead of a text field.

**Review by JMGbaum — DISMISSED** [review]

**Review by JMGbaum — DISMISSED** [review]

File: test/controllers/ingredient_items_controller_test.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 class IngredientItemsControllerTest < ActionDispatch::IntegrationTest
   test "should get new" do
-    get ingredient_items_new_url
+    get ingredient_items_show_path
```
Comment (JMGbaum) [inline test/controllers/ingredient_items_controller_test.rb:5]: Didn't this path not work?

Comment (beckystar4) [reply test/controllers/ingredient_items_controller_test.rb:5]: I haven't fixed it yet.

**Review by JMGbaum — APPROVED** [review]

**Comment by beckystar4** [comment]
[Create login page](https://trello.com/c/l4RoogM5/9-create-login-page)

**Comment by beckystar4** [comment]
[Password validation](https://trello.com/c/RD17jOtX/28-password-validation)

**Comment by beckystar4** [comment]
[Backend Login Implementation](https://trello.com/c/Oc3UQPiw/34-backend-login-implementation)

**Pull request merged**